.circleci/config.yml

```diff
@@ -491,12 +516,14 @@ jobs:
           at:  /go
       - setup_remote_docker:
           docker_layer_caching: true
+      - run: make istio-archive
       - run:
           command: |
             if [ ! -z "${DOCKER_USER}" ] ; then
               echo "Pushing docker images"
               docker login -u ${DOCKER_USER} -p ${DOCKER_PASS}
-              make docker push
+              # The xenail debug image is built first, will be used in the next steps
+              make docker.push.basedebug docker.all push
```

Comment:
docker.basedebug.push to be consistent with docker.pilot and other docker.<image> rules?